PlanetNode/GraphTypes/PlanetNodeMutation.cs

```diff
@@ -34,19 +35,15 @@ public PlanetNodeMutation(BlockChain<PolymorphicAction<PlanetAction>> blockChain
                 string privateKeyHex = context.GetArgument<string>("privateKeyHex");
 
                 PrivateKey privateKey = PrivateKey.FromString(privateKeyHex);
-                TransferAsset action = new TransferAsset(
-                    privateKey.ToAddress(),
+                var action = new Transfer(
```

Comment:
It seems that we don't need `TransferAsset` action anymore. if so, how about removing it?

Reply:
I also considered to do that, but I decided to leave it yet for an example of custom actions.  However, if you find it's still not worth I'm going to get rid of it.  What do you think about this?

Reply:
Ah, yeah. that makes sense... then, lefting comments on `TransferAsset` as reminder would be nice. 
I'm going to create an issue for new example action. (I guess `Burn` can be enough in that case).

Reply:
Left a comment (I amended the commit).  Please take a look again?